spec/models/measure_condition_spec.rb

```diff
@@ -6,4 +6,16 @@
   describe '#requirement' do
     it { expect(condition.requirement).to be_html_safe }
   end
+
+  describe 'attributes' do
+    it {
+      expect(condition).to respond_to(:condition,
```

Comment:
We could use a shared example to achieve this/make this a validation of all api entity models:

```ruby
RSpec.shared_examples 'a resource that has attributes' do |attributes|
  subject(:resource) { described_class.new(attributes) }

  describe 'implemented attributes' do
    attributes.each do |attribute_key, attribute_value|
      it "defines an accessor for #{attribute_key}" do
        expect(resource.public_send(attribute_key)).to eq(attribute_value)
      end
    end
  end
end
```